src/chrome/content/rules/Nic.ir.xml

```diff
@@ -1,18 +1,11 @@
 <!--
-	Nonfunctional hosts in *.nic.ir:
-		-w whois.nic.ir (e)
-
-	e: expired certificate
-	h: http redirect
-	m: certificate mismatch
-	r: connection refused
-	s: self-signed certificate
-	t: timeout on https
+	Expired:
+		- tools
+		- whois
 -->
-<ruleset name="Nic.ir">
+<ruleset name="Nic.ir" default_off="cert-chain">
```

Comment:
@pipboy96 this does only affect `rrmt.nic.ir`.

Reply:
There's other breakage, not sure if splitting this into two rulesets and making one default_off and whitelisting the other is worth it.

Reply:
Feel free to make a follow-up PR if you want to. I will merge it but I don't want to make one myself.

Reply:
#17634